networkx/algorithms/bipartite/projection.py

```diff
@@ -337,10 +337,10 @@ def overlap_weighted_projected_graph(B, nodes, jaccard=True):
     >>> G = bipartite.overlap_weighted_projected_graph(B, [0, 2, 4])
     >>> print(G.nodes())
     [0, 2, 4]
-    >>> print(G.edges(data=True))
+    >>> print(list(G.edges(data=True)))
```

Comment:
I thought of doing a final manual cleanup of the codebase on issues like using print() in doctests which is present in a lot of places, and low priorities like uniformly using decorators all over the codebase before the release of 2.0 .